Challenge 5

shloksuman opened the pull request:
Added my details and created **PULL REQUEST** from my forked repo's second branch ( shloksuman-details) to scaler's main repo main branch

Mrjoy832 reviewed (changes requested):
Checks Failed
md file wrong content
do squashing(too many commits)

shloksuman commented:
Sir ,
I have rectified the above mentioned mistakes.
1. Corrected Md file content
2. Passed the DCO Checks
3. Squashed the commits

On Wed, Sep 14, 2022 at 9:58 AM TRIDIB BAG ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> Checks Failed
> md file wrong content
> do squashing(too many commits)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/scaleracademy/scaler-september-open-source-challenge/pull/793#pullrequestreview-1106736617>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASRMRKG5J3U7DVCRAN5O3ELV6FIARANCNFSM6AAAAAAQFFAVKM>
> .
> You are receiving this because you authored the thread.Message ID:
> <scaleracademy/scaler-september-open-source-challenge/pull/793/review/1106736617
> @github.com>
>

Mrjoy832 commented:
I am busy Today ...extremely sorry ...I have an exam tomorrow...I will clear this by tomorrow

shloksuman commented:
SIr , my tokens for completing milestone 1 have not yet been credited to my
account. It still shows only 30 tokens.You have even verified my challenges
and gave thumbs up. Please check the matter.
Github Issue Link :
https://github.com/scaleracademy/scaler-september-open-source-challenge/issues/232
Github PR Link :
https://github.com/scaleracademy/scaler-september-open-source-challenge/pull/793
Discord username : shloksuman#3978


On Thu, Sep 15, 2022 at 9:47 PM Shlok Suman ***@***.***> wrote:

> Sure, no problem.
>
> On Thu, Sep 15, 2022 at 7:55 PM TRIDIB BAG ***@***.***>
> wrote:
>
>> I am busy Today ...extremely sorry ...I have an exam tomorrow...I will
>> clear this by tomorrow
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/scaleracademy/scaler-september-open-source-challenge/pull/793#issuecomment-1248177865>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ASRMRKFV2OFCWN6BH4BEP23V6MWT7ANCNFSM6AAAAAAQFFAVKM>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> <scaleracademy/scaler-september-open-source-challenge/pull/793/c1248177865
>> @github.com>
>>
>

shloksuman commented:
Challenge 22 : Host a static website using Github Pages

Screenshot : 

![day 22 host a website](https://user-images.githubusercontent.com/77777064/191775535-33f52aec-3f8c-4375-abd0-5139ca7142d8.png)

[**Repo**](https://github.com/shloksuman/scalerstatic)
[**Github Page Link**](https://shloksuman.github.io/scalerstatic/)

kuraakhilesh8230 commented:
too many commits pls correct that by tomorrow

shloksuman commented:
Thank you sir for your response. I have reduced the number of commits.
Github PR link :
https://github.com/scaleracademy/scaler-september-open-source-challenge/pull/793/
(screenshot also attached)

On Sat, Sep 24, 2022 at 6:23 PM Kura Akhilesh ***@***.***>
wrote:

> too many commits pls correct that by tomorrow
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/scaleracademy/scaler-september-open-source-challenge/pull/793#issuecomment-1256962069>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASRMRKA34OBRPW3NXD2RBKLV732TRANCNFSM6AAAAAAQFFAVKM>
> .
> You are receiving this because you authored the thread.Message ID:
> <scaleracademy/scaler-september-open-source-challenge/pull/793/c1256962069
> @github.com>
>

shloksuman commented:
> 

Done.
Reduced the number of commits.

kuraakhilesh8230 commented:
congrats you have completed the milestone-2 successfully and you are good to go🎉
